crates/turborepo-lib/src/commands/link.rs

```diff
@@ -159,6 +178,12 @@ fn select_team<'a>(teams: &'a [Team], user_display_name: &'a str) -> Result<Sele
     }
 }
 
+#[cfg(test)]
+fn should_link(_: &CommandBase, _: &str) -> Result<bool> {
+    Ok(true)
+}
+
+#[cfg(not(test))]
```

Comment:
Not from this PR, but I'm noticing that we're not disabling style for the bolded and grey'd out text in this function